package.json

```diff
@@ -25,16 +25,19 @@
   "dependencies": {
     "node-sass-chokidar": "0.0.3",
     "npm-run-all": "^4.1.2",
-    "react": "^16.1.1",
+    "prop-types": "^15.6.1",
+    "react": "^16.3.0-alpha.2",
     "react-app-rewire-styled-components": "^3.0.0",
     "react-app-rewire-svg-react-loader": "github:codebandits/react-app-rewire-svg-react-loader",
     "react-app-rewired": "^1.3.6",
-    "react-dom": "^16.1.1",
+    "react-dom": "^16.3.0-alpha.2",
```

Comment:
Why do we need the alpha release?

Reply:
I'll revert that before finishing, was just trying the context API but don't think it will be needed